### PR TITLE
do flush in io-redicret

### PR DESCRIPTION
**Pull request by behrica**

### CODE REVIEW

File: src/libpython_clj2/python/io_redirect.clj [hunk 1]
```diff
@@ -32,7 +42,9 @@
                 (py-ffi/py-none))
               {:arg-converter identity})
      "flush" (py-class/make-tuple-instance-fn
-              (constantly (py-ffi/py-none)))
+              (fn [self] (.flush *err*))) ; no idea how to pass and use `writer-var` here
```
Comment (cnuernber) [inline src/libpython_clj2/python/io_redirect.clj:45]: It is used 4 lines above this one - 

```clojure
(fn [self & args] (.flush (self->writer self)) (py-ffi/py-none))
{:arg-converter identity} ;;avoid paying anything for argument conversion
```

Comment (cnuernber) [reply src/libpython_clj2/python/io_redirect.clj:45]: In my tests at the repl that will print things whether it flushes or not:

```clojure
libpython-clj2.python.io-redirect> (.write ^Writer *err* "#")
#nil
```

**Comment by behrica** [comment]
This code behaves different with/without the above change:

```
(py/run-simple-string "for i in range(10):\n        sleep(1)\n        sys.stderr.write('#')\n        sys.stdout.flush()")
```

**Comment by behrica** [comment]
I can force the printing afterwards like this:

```
(.flush *err*)
```

**Comment by behrica** [comment]
So both together do work:

```
(py/run-simple-string "for i in range(10):\n        sleep(1)\n        sys.stderr.write('#')\n        sys.stderr.flush()")
(.flush *err*)
```

Except that it prints it "to late", not every second.

**Comment by cnuernber** [comment]
Got it.  I have the same change in my codebase - do you mind if I use that as opposed to this PR?  This PR as it stands is somewhat incorrect and if literally the only thing that needs to be added is the implementation of flush then I think it is simpler if I do it.

**Comment by behrica** [comment]
Fine for me.

**Pull request closed without merging**